proyecto/src/main/java/com/proyecto/proyecto/ControladorUsuarios.java

```diff
@@ -32,5 +32,19 @@ public String eliminar(@PathVariable("id") int idUsuario, RedirectAttributes att
         return "redirect:/usuarios/index";
     }
 
+    @GetMapping("/unlock/{id}")
+	public String activar(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {		
+    	servicioUsuarios.activar(idUsuario);
+		attributes.addFlashAttribute("msg", "El usuario fue activado y ahora tiene acceso al sistema.");		
+		return "redirect:/usuarios/index";
+	}
+
+    @GetMapping("/lock/{id}")
+	public String bloquear(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {		
+		servicioUsuarios.bloquear(idUsuario);
+		attributes.addFlashAttribute("msg", "El usuario fue bloqueado y no tendra acceso al sistema.");		
+		return "redirect:/usuarios/index";
+	}
+
```

Comment:
#should La indentación en estos cambios está rara si se la compara con las que ya existían en el archivo. Acomodar, por favor!

Si en el archivo en la computadora lo ves bien, es posible que estés usando tabulaciones en lugar de espacios para la indentación. Hay que cambiar eso en el editor.